algopy_testing/src/algopy/utils.py

```diff
@@ -59,3 +59,19 @@ def as_bytes(value: object, *, max_size: int = MAX_BYTES_SIZE) -> bytes:
     if len(bytes_value) > max_size:
         raise ValueError(f"expected value length <= {max_size}, got: {len(bytes_value)}")
     return bytes_value
+
+
+def as_string(value: object, *, max_size: int = MAX_STRING_SIZE) -> str:
```

Comment:
I think we can remove the max_size parameter also, it's not used below and I don't think we will have need of having different sized String's beyond the normal max